### PR TITLE
use external `{BitSet, VecMap}`

### PR DESCRIPTION
**Pull request by apasel422**

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @bluss

(I've picked a reviewer for you, use r? to override)

**Comment by bluss** [comment]
Looks like this should be easy to build on stable now? (Do you want to merge it anyway? lgtm)

File: Cargo.toml [hunk 1]
```diff
@@ -15,3 +15,8 @@ homepage = "https://github.com/contain-rs/ordered_iter"
 documentation = "https://contain-rs.github.io/ordered_iter/ordered_iter"
 keywords = ["data-structures"]
 readme = "README.md"
+
+[dependencies]
+bit-set = "*"
+bit-vec = "*"
+vec_map = "*"
```
Comment (bluss) [inline Cargo.toml]: best to have these pin a 0.x version, if we're going stable.

**Comment by apasel422** [comment]
@bluss This should be good to go now.

**Comment by bluss** [comment]
nice!

**Pull request merged**